receiver/mongodbreceiver/metadata.yaml

```
@@ -1 +1,143 @@
 name: mongodbreceiver
+
+attributes:
+  database:
+    description: The name of a database.
+  memory_type:
+    description: The type of memory used.
+    enum:
+      - resident
+      - virtual
+      - mapped
+      - mappedWithJournal
```

```suggestion
      - mapped_with_journal
```
Is "mapped with journal" a domain specific term? If not, is there a more concise alternative that would work? (e.g. `journaled`)

---

Hmm so this value is 
> the amount of mapped memory, in megabytes (MB), including the memory used for journaling

So the initial value from the API was mapped + journaled memory. I imagine if we wanted to just report the explicit memory used for journaling, we could take the difference?